File: exercises/beecrowd/1044/1044.go
// https://judge.beecrowd.com.br/judge/pt/problems/view/1044
package main

import (
	"fmt"
)

func main() {
	var a, b int

	fmt.Scanf("%d %d", &a, &b)

	if a%b == 0 || b%a == 0 {
		fmt.Println("Sao Multiplos")
	} else {
		fmt.Println("Nao sao Multiplos")
	}
}

File: exercises/beecrowd/1044/1044.ts
// https://judge.beecrowd.com/pt/problems/view/1044
import * as fs from "fs";

function main(): void {
	const input = fs.readFileSync("/dev/stdin", "utf8");
	const lines = input.split(/\r?\n/);

	const line = String(lines.shift()).split(' ');

	const a = Number(line[0])
	const b = Number(line[1])

	if (a%b == 0 || b%a == 0) {
		console.log("Sao Multiplos")
	} else {
		console.log("Nao sao Multiplos")
	}
}

main();
